store/root/store.go

```diff
@@ -198,17 +184,37 @@ func (s *Store) GetBranchedKVStore(_ string) store.BranchedKVStore {
 	return s.rootKVStore
 }
 
-func (s *Store) loadVersion(v uint64, upgrades any) error {
+func (s *Store) LoadLatestVersion() error {
+	lv, err := s.GetLatestVersion()
+	if err != nil {
+		return err
+	}
+
+	return s.loadVersion(lv, nil)
+}
+
+func (s *Store) LoadVersion(version uint64) (err error) {
+	return s.loadVersion(version, nil)
+}
+
+func (s *Store) loadVersion(v uint64, _ *store.StoreUpgrades) error {
```

Comment:
It is a WIP? couldn't see the call of `loadVersion` with upgrades.

Reply:
This implementation doesn't support store upgrades since it's all a single tree/store key.